projects/cra-to-nx/src/lib/cra-to-nx.ts

```diff
@@ -140,6 +143,10 @@ export async function createNxWorkspaceForReact(options: Record<string, any>) {
 
   setupTsConfig(reactAppName);
 
+  output.log({ title: '🔎 Making sure that workspace.json exists' });
+
+  createWorkspaceJson(reactAppName, options.e2e);
```

Comment:
We don't need this part because latest Nx doesn't need `workspace.json` for minimal setup.

Reply:
Ok, removed it! :D